### DISCUSSION TITLE
Supress output output of $this->outp_throw() in mysqli();

### DISCUSSION THREAD

**Discussion by mariobernheim** [ADOdb/ADOdb#1048] [Q&A] [answered]
Hello - this is on Adodb 5.22.7 and php 8.3 - 

in adodb-mysqli.inc.php
```
			$stmt = $this->_connectionID->prepare($sql);
			if ($stmt === false)
			{
				$this->outp_throw(
					"SQL Statement failed on preparation: " . htmlspecialchars($sql) . "'",
					'Execute'
				);
				return false;
			}
			
```

What's the proper way to supress this output - setting error_reporting to 0 is not supressing it.

**Comment by joebordes** [comment, marked as answer]
probably something like this:

```php
function adodbOutput($msg, $newline) {
	// do nothing
}
define('ADODB_OUTP', 'adodbOutput');
```

https://adodb.org/dokuwiki/doku.php?id=v5:reference:logging

HTH

**Comment by dregad** [comment]
In addition to what @joebordes wrote, I suggest you also read https://adodb.org/dokuwiki/doku.php?id=v5:userguide:error_handling.

**Reply by mariobernheim** [reply]
thank you both
